_providers/zoho.md

```diff
@@ -0,0 +1,26 @@
+---
+name: Zoho Mail
+domains: 
+  - zohomail.eu
```

Comment:
is that really only zohomail.eu? according to https://www.zoho.com/mail/help/imap-access.html#EnableIMAPAccess at least addresses as username@zoho.com exist.

Reply:
@r10s I addeed zoho.com addresses

Reply:
@r10s When will it be mergered?

Reply:
i was just waiting for ci, merging it now :)

it will then be available at https://providers.delta.chat and in the apps with the next release.